exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 let studentCount = 16;
 let mentorCount = 9;
-let moreStudentsThanMentors; // finish this statement
+let moreStudentsThanMentors = (studentCount > mentorCount); // finish this statement
 
 let roomMaxCapacity = 25;
-let enoughSpaceInRoom; // finish this statement
+let enoughSpaceInRoom = ((studentCount + mentorCount) === roomMaxCapacity); // finish this statement
```

Comment:
How about when (studentCount + mentorCount) is More Than the roomMaxCapacity.
Your test does not appear to cater for it.